src/evo/providertx.h

```diff
@@ -149,6 +149,10 @@ class CProUpServTx
             // unknown version, bail out early
             return;
         }
+        if (obj.nVersion != BASIC_BLS_VERSION && obj.nType == MnType::HighPerformance) {
+            // for HPMN should be only BASIC bls, bail out early
+            return;
+        }
```

Comment:
5989de876f34adcc672a9dfe7f8fcc270b507fb9 should probably be it's own PR... maybe even a backport?

Reply:
why do backporting only this commit but not other commits in this PR? What's a difference?